### PR TITLE
Add CombinatorialSpaces.jl to package list

### PR DESCRIPTION
**Pull request by mehalter**
Signed-off-by: Micah Halter <micah@balena.io>

### CODE REVIEW

**Comment by mehalter** [comment]
@epatters I don't know exactly how built out CombinatorialSpaces.jl is atm so you might want to go in and adjust the package status accordingly

**Comment by epatters** [comment]
That is accurate but honestly I wonder whether we should remove that column and just have name and perhaps slightly longer description.

File: page/index.md [hunk 1]
```diff
@@ -16,12 +16,13 @@
...
+| [`AlgebraicDynamics.jl`](https://algebraicjulia.github.io/AlgebraicDynamics.jl/dev/)     | Build dynamical systems compositionally                            |         <a class="github-button" href="https://github.com/AlgebraicJulia/AlgebraicDynamics.jl" data-size="large" aria-label="View on GitHub">AlgebraicDynamics.jl on GitHub</a></br> Functionality is mostly built-out, but the API may change substantially.         |
+| [`CombinatorialSpaces.jl`](https://algebraicjulia.github.io/CombinatorialSpaces.jl/dev/) | Simplicial sets and other combinatorial models of geometric spaces | <a class="github-button" href="https://github.com/AlgebraicJulia/AlgebraicDynamics.jl" data-size="CombinatorialSpaces" aria-label="View on GitHub">CombinatorialSpaces.jl on GitHub</a></br> Functionality is mostly built-out, but the API may change substantially. |
```
Comment (epatters) [inline page/index.md]: Link in GitHub button points to AlgebraicDynamics.

Comment (mehalter) [reply page/index.md]: So it does.... sorry about that! Fixed in latest push

**Comment by mehalter** [comment]
Yeah that would make sense as well to remove that column all together and add the information to description

**Comment by epatters** [comment]
Great, I'll merge this PR and we can do that separately.

**Pull request merged**